vNext/src/ClientGenerator/ClientGenerator.csproj

```diff
@@ -18,21 +24,73 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <Compile
-      Include="..\..\..\src\ClientGenerator\**\*.cs"
-      Exclude="
-        ..\..\..\src\ClientGenerator\obj\**\*.cs;
-        ..\..\..\src\ClientGenerator\bin\**\*.cs;
-        ..\..\..\src\ClientGenerator\Properties\**\*.cs" />
+    <Compile Include="..\..\..\src\ClientGenerator\**\*.cs" Exclude="..\..\..\src\ClientGenerator\obj\**\*.cs;..\..\..\src\ClientGenerator\bin\**\*.cs;..\..\..\src\ClientGenerator\Properties\**\*.cs" />
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="System.Runtime" Version="4.3.0" />
+    <PackageReference Include="System.Runtime" Version="4.3.0">
+      <PrivateAssets>all</PrivateAssets>
+    </PackageReference>
   </ItemGroup>
 
   <ItemGroup>
-    <ProjectReference Include="..\Orleans.PlatformServices\Orleans.PlatformServices.csproj" />
-    <ProjectReference Include="..\Orleans\Orleans.csproj" />
-    <ProjectReference Include="..\OrleansCodeGenerator\OrleansCodeGenerator.csproj" />
+    <ProjectReference Include="..\Orleans.PlatformServices\Orleans.PlatformServices.csproj">
+      <PrivateAssets>all</PrivateAssets>
+    </ProjectReference>
+    <ProjectReference Include="..\Orleans\Orleans.csproj">
+      <PrivateAssets>all</PrivateAssets>
+    </ProjectReference>
+    <ProjectReference Include="..\OrleansCodeGenerator\OrleansCodeGenerator.csproj">
+      <PrivateAssets>all</PrivateAssets>
+    </ProjectReference>
   </ItemGroup>
+
+  <ItemGroup>
+    <None Include="..\Orleans.SDK.targets">
+      <Pack>true</Pack>
+      <PackagePath>build\$(PackageId).targets</PackagePath>
+      <Visible>false</Visible>
+    </None>
+    <None Include="$(OutDir)/ClientGenerator.exe.config">
```

Comment:
It's a little unfortunate that these need to be included manually instead of the references being automatically included in the package, but so be it, I suppose.